models/offer.go

```diff
@@ -38,6 +42,37 @@ const enabledOffers = `
 		AND offers.enabled = true
 `
 
+func buildScope(enabledOffers string, filterAttrs map[string]string) string {
+	subQueries := []string{enabledOffers}
+	for k, v := range filterAttrs {
+		// TODO: Possible SQL injection
```

Comment:
Unfortunately, this scenario is similar to this issue:
https://github.com/golang/go/issues/18478

And wasn't able to find a golang lib that I could use only to escape the arguments, hopefully these will be created soon. (Or I might try to extract them sometime).